README.md

```diff
@@ -1,181 +1,78 @@
-any directory containing ".git", "setup.py",  "setup.cfg", "pyproject.toml",
-or "requirements.txt")
-5. See [Keybindings and completion](#Keybindings-and-completion) for mapping useful functions and enabling
-omnifunc completion
-
-## Built-in commands
-
-* `:LspInfo` shows the status of active and configured language servers.
-
-The following support tab-completion for all arguments:
-
-* `:LspStart <config_name>` Start the requested server name. Will only succesfully start if the command detects a root directory matching the current config. Pass `autostart = false` to your `.setup{}` call for a language server if you would like to launch clients solely with this command. Defaults to all servers matching current buffer filetype.
-* `:LspStop <client_id>` Defaults to stopping all buffer clients. 
-* `:LspRestart <client_id>` Defaults to restarting all buffer clients.
-
-## Usage
-
-**All provided examples are in Lua,** see `:help :lua-heredoc` to use Lua from your init.vim,
-or the quickstart above for an example of a lua heredoc.
-
-Each config provides a `setup()` function to initialize the server with reasonable default
-initialization options and settings, as well as some server-specific commands. This is 
-invoked in the following form, where `<server>` corresponds to the language server name
-in [CONFIG.md](CONFIG.md).
-
-`setup()` takes optional arguments <config>, each of which overrides the respective 
-part of the default configuration. The allowed arguments are detailed [below](#setup-function).
-```lua
-require'lspconfig'.<server>.setup{<config>}
+3. Create a project, this project must contain a file matching the root directory trigger. See [Automatically launching language servers](#Automatically-launching-language-servers) for additional info.
+```bash
+mkdir test_python_project
+cd test_python_project
+git init
+touch main.py
 ```
+4. Launch neovim, the language server will now be attached and providing diagnostics (see `:LspInfo`)
+`nvim main.py`
+5. See [Keybindings and completion](#Keybindings-and-completion) for mapping useful functions and enabling omnifunc completion
 
-### Example: using the defaults
+## Automatically launching language servers
 
-To use the defaults, just call `setup()` with an empty `config` parameter.
-For the `gopls` config, that would be:
+In order to automatically launch a language server, lspconfig searches up the directory tree from your current buffer to find a file matching the `root_dir` pattern defined in each server's configuration.  For [pyright](CONFIG.md#pyright), this is any directory containing ".git", "setup.py",  "setup.cfg", "pyproject.toml", or "requirements.txt").
 
-```lua
-require'lspconfig'.gopls.setup{}
-```
-
-### Example: override some defaults
+Language servers require each project to have a `root` in order to provide completion and search across symbols that may not be defined in your current file, and to avoid having to index your entire filesystem on each startup.
```

Comment:
If you change the heading then this sentence should be the first, to introduce the key concept of the `root` before going into detail.